samples/zigbee/light_switch/boards/nrf5340dk_nrf5340_cpuapp_fota.conf

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Use software cryptography on nRF5340
+CONFIG_ZIGBEE_USE_SOFTWARE_AES=y
+
+# Networking
+CONFIG_MPSL=n
+
+
+#######################
+# Zigbee FOTA overlay #
+#######################
+
+# Allow for storing two images in MCUboot partitions
+CONFIG_UPDATEABLE_IMAGE_NUMBER=2
+
+# Store new images inside external flash
+CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=y
+
+# Configure QSPI for external flash
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
+
+CONFIG_BOOTLOADER_MCUBOOT=y
```

Comment:
Technically, these three options are already enabled by the Zigbee FOTA library:
CONFIG_BOOTLOADER_MCUBOOT
CONFIG_MCUMGR
CONFIG_DFU_TARGET

Reply:
Removed.